VAMobile/src/components/CommonErrorComponents/DowntimeError.tsx

```diff
@@ -31,9 +30,12 @@ const DowntimeError: FC<DowntimeErrorProps> = ({ screenID, overrideFeatureName }
     mb: theme.dimensions.contentMarginBottom,
   }
   const { downtimeWindowsByFeature } = useSelector<RootState, ErrorsState>((state) => state.errors)
-  const feature = DowntimeScreenIDToFeature[screenID]
-  const featureName = overrideFeatureName ? overrideFeatureName : downtimeWindowsByFeature[feature]?.featureName
-  const endTime = downtimeWindowsByFeature[feature]?.endTime.toFormat('fff')
+  const feature = ScreenIDToDowntimeFeature[screenID]
+  const downtimeWindow = feature ? downtimeWindowsByFeature[feature] : null
```

Comment:
This type of pattern was a recurring pain point I experienced with TS. I would expect that there's probably a better pattern for this.